### PR TITLE
Add manual download link to upgrade wizard

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -599,6 +599,7 @@ en:
           will be asked to upload it later on in the process.
           After you downloaded and saved the upgrade data click 'Continue Upgrade Process'.
         blockui_message: 'Aborting Upgrade...'
+        link: 'Manual download link:'
```
Comment (vuntz) [inline crowbar_framework/config/locales/crowbar/en.yml]: Why is it "manual"? Or do you mean "Direct download link"?

(tbh, I'm not totally sure how it looks like, maybe a screenshot would help :-))

**Comment by rhafer** [comment]
screen

It looks like this. But the link doesn't work for me.

**Comment by vuntz** [comment]
Thanks. So that'd be "Direct download link". And I'd put it after the button :-)

**Comment by rsalevsky** [comment]
@rhafer The button and the link should point to the same url. Can you check this?

**Comment by jsuchome** [comment]
@vwallfahrer @rsalevsky button leads correctly to `installer/upgrade/file`, but the link to `installer/upgrade/download`

**Comment by vwallfahrer** [comment]
bildschirmfoto vom 2016-02-10 09 41 17

It looks like this now, @kwwii.

**Comment by vwallfahrer** [comment]
I talked to @kwwii and the link has to be above the button.

**Comment by aplanas** [comment]
@vwallfahrer ask @kwwii also about the margins. After that: +1

**Comment by vuntz** [comment]
> I talked to @kwwii and the link has to be above the button.

Ah, sorry.

**Comment by rsalevsky** [comment]
@kwwii Approved this change.

+1

**Comment by vuntz** [comment]
But does the link work? @rhafer said it wasn't working for him.

**Comment by tboerger** [comment]
I would suggest to put it into a text input at the side of the button :)

**Pull request merged**